package.json

```diff
@@ -75,7 +76,7 @@
     "react-router-dom": "^4.3.1",
     "react-youtube": "^7.8.0",
     "screenfull": "^3.3.2",
-    "super-hands": "github:mozillareality/aframe-super-hands-component#feature/drawing",
+    "super-hands": "github:mozillareality/aframe-super-hands-component#5a7ffe9805870f4821e6e636486a7cce01c6cf64",
```

Comment:
Same preference for pointing at a meaningfully named branch rather than a commit sha. Not sure if this preference is justified, to be honest. But figured I'd mention it anyway.